NF.csv

```diff
@@ -724,7 +725,7 @@ compoundName,"Common name for a compound, e.g. “Selumetinib” (https://pubche
 compoundDose,A dose quantity for the treatment compound. To be used with compoundDoseUnit.,,compoundDoseUnit,FALSE,Sage Bionetworks,sageCommunity,,,,compoundDose,Property,Experiment,annotationProperty,experimentProperty,,,
 compoundDoseUnit,A unit associated with the value(s) in compoundDose.,,,FALSE,Sage Bionetworks,sageCommunity,,,,compoundDoseUnit,Property,Experiment,annotationProperty,experimentProperty,,,
 InChIKey,"A unique hash key for a compound, e.g. for “Selumetinib”, the key is CYOHGALHFOKKQC-UHFFFAOYSA-N (https://pubchem.ncbi.nlm.nih.gov/compound/10127622#section=InChI-Key). This is a more reliable identifier than the compound name and should be used if available.",,,FALSE,Sage Bionetworks,sageCommunity,,,,InChIKey,Property,Experiment,annotationProperty,experimentProperty,,,
-genePerturbationType,Specific way in which a single gene was perturbed in a sample,"overexpression, knockdown, knockout, Not Applicable",,FALSE,Sage Bionetworks,cancer,,,,genePerturbationType,Property,Experiment,annotationProperty,,Gene_Perturbation,,
+genePerturbationType,Specific way in which a single gene was perturbed in a sample,"overexpression, knockdown, knockout, control, Not Applicable",,FALSE,Sage Bionetworks,cancer,,,,genePerturbationType,Property,Experiment,annotationProperty,,Gene_Perturbation,,
```

Comment:
Use "non-targeting control"?

Reply:
Made change that I suggested in https://github.com/nf-osi/nf-metadata-dictionary/pull/182/commits/aab5ca9c70b184a103add19b643514e54edf27e1...

Reply:
Yup...that is what I should be using. Thanks!